navigation/tracking/scripts/plots.py

```diff
@@ -0,0 +1,262 @@
+import sys
+
+sys.path.insert(0, "/home/hannahcl/Documents/vortex/monkey_tracking/data_generation")
```

Comment:
☹️ 

Modules outside of a package can be set up with a setup.py and installed with pip, but I don't think theres a guide for that on the wiki yet. This hack should at least be addressed in some way, potentially just with an installation guide or pointer to the repo containing the data generator.